src/VisualStudio/Core/Def/Implementation/EditAndContinue/VsENCRebuildableProjectImpl.cs

```diff
@@ -868,6 +880,8 @@ public int ExitBreakStateOnPE()
                     {
                         Debug.Assert(s_breakStateProjectCount == s_debugStateProjectCount);
 
+                        _encService.OnBeforeDebuggingStateChanged(DebuggingState.Break, DebuggingState.Run);
```

Comment:
ExitBreakStateOnPE is called once per project, but the check "if (_encService.EditSession != null)" on line 879 ensures that this block is only executed the first time, so I don't need to add my own check here.